cms/templates/admin/cms/page/tree/menu.html

```diff
@@ -260,7 +260,7 @@
     '
     >
 
-    {% language preview_language %}
+    {% language LANGUAGE_CODE %}
```

Comment:
So far the CMS has supported changing the display title of the page via this ``?language`` parameter, this change effectively reverts that. I don't agree on breaking this backwards compatibility.